docs/src/pages/customization/overrides/overrides.md

```diff
@@ -17,7 +17,7 @@ You might need to change the style of a component in some very specific situatio
 The first way to override the style of a component is to use **class names**.
 Every component provides a `className` property which is always applied to the root element.
 
-In this example, its using the [`withStyles()`](/customization/css-in-js/#withstyles-styles-options-higher-order-component) higher-order
+This example is using the [`withStyles()`](/customization/css-in-js/#withstyles-styles-options-higher-order-component) higher-order
```

Comment:
```suggestion
This example uses the [`withStyles()`](/customization/css-in-js/#withstyles-styles-options-higher-order-component) higher-order
```

Reply:
You might need to change the style of a component in some very specific situation, for which you have the following solutions available:

---
I think this should be changed, I haven't found a sentence I like but I've suggested a few

You might need to change the style of a component for a one-off situation, for which you have the following solutions available:

You might need to change the style of a component for a specific implementation, for which you have the following solutions available:

Reply:
You can choose [any other styling solution](/guides/interoperability/), or even plain CSS to create the styles, but be sure to
consider the [CSS injection order](/customization/css-in-js/#css-injection-order), as the CSS injected into the DOM
by Material-UI to style a component has the highest specificity possible since the `<link>` is injected at the bottom
of the `<head />` to ensure the components always render correctly.

---

I think `by Material-UI` disrupts the sentence here I would add either commas or parenthesis around it not sure which. Not sure if it's worth changing this